src/commands/meal.py

```diff
@@ -3,29 +3,24 @@
     DINNER
 )
 from src.util.messages import no_menu_msg, menu_msg
-from src.util.util import parse_menu
-from src.database.database import connect
-from src.database.queries import menu_query
-import psycopg2.extras
+from src.util.util import parse_menu, get_menu_from_db, get_hidden_cuisines
 from datetime import date
 
 
 def handle_menu(meal):
     assert meal == BREAKFAST or meal == DINNER, "Meal input is incorrect."
 
     def get_breakfast_or_dinner_menu(update, context):
-        """Send the user menu"""
-        # get menu from database
-        conn = connect()
-        cur = conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
-        cur.execute(menu_query(meal), (date.today(),))
-        data = cur.fetchone()
+        # send the user menu
+        menu = get_menu_from_db(meal, date.today())
 
-        if data is None:  # if no menu, reply with no menu message
-            update.message.reply_text(no_menu_msg(meal))
+        hidden_cuisines = get_hidden_cuisines(update.effective_chat.id)
+
+        if menu is None:  # if no menu, reply with no menu message
+            context.bot.send_message(chat_id=update.effective_chat.id, text=no_menu_msg(meal))
```

Comment:
Should we standardise to use reply_text? It's a shortcut command so we don't need to include the chat_id every time.

Reply:
for callback_query, reply_text doesn't seem to work coz the message is out of context. In this case, we need to use the bot in the context to actually send the message